motif_inference/unite_motifs_of_biological_condition.py

```diff
@@ -108,7 +108,7 @@ def unite_clusters(motif_inference_output_path, meme_file, biological_condition,
     with open(sorted_clusters_to_combine_path, 'w') as f:
         for cluster_names in clusters_to_combine:
             num_of_samples_per_cluster = get_samples_number_build_cluster(cluster_names) 
-            if num_of_samples_per_cluster >= min_number_samples_build_cluster:
+            if int(num_of_samples_per_cluster) >= int(min_number_samples_build_cluster):
```

Comment:
This might work but it's wrong.
`num_of_samples_per_cluster` is already an integer.
`min_number_samples_build_cluster` should be an integer, if you look closely it's type in command argument is wrong, a simple change there would fix the entire issue